server/pulp/server/async/scheduler.py

```diff
@@ -209,27 +159,29 @@ def spawn_pulp_monitor_threads(self):
         """
         Start two threads that are important to Pulp. One monitors workers, the other, events.
 
-        Two threads are started, one that uses EventMonitor, and handles all Celery events. The
-        second, is a WorkerTimeoutMonitor thread that watches for cases where all workers
+        A WorkerTimeoutMonitor thread is started that watches for cases where all workers
         disappear at once.
 
-        Both threads are set with daemon = True so that if the main thread exits, both will
-        close automatically. After being daemonized they are each started with a call to start().
+        The thread is set with daemon = True so that if the main thread exits, it will close
+        automatically.
 
         This method should be called only in certain situations, see docstrings on the object for
         more details.
         """
-        # start monitoring events in a thread
-        event_monitor = EventMonitor()
-        event_monitor.daemon = True
-        event_monitor.start()
         # start monitoring workers who may timeout
         worker_timeout_monitor = CeleryProcessTimeoutMonitor()
         worker_timeout_monitor.daemon = True
         worker_timeout_monitor.start()
 
-    @staticmethod
     def call_tick(self, celerybeat_name):
+        """
+        Call the superclass tick method and log a debug message.
+
+        :param celerybeat_name: hostname of the celerybeat instance
+        :type  celerybeat_name: basestring
+        :return:                seconds until the next tick
```

Comment:
I usually separate arguments from return params with a line of whitespace. Just a suggestion.